### ISSUE TITLE
Larger Space Option

### ISSUE THREAD

**Issue by devoxel** [futo-org/android-keyboard#964]
### Use Cases

I am constantly hitting the button to the right of the space when i mean to his space, when i type quickly. It looks like "this. Is. My. Search". I guess my thumb is stupid. 

### Proposal

I'd like a layout with a spacebar that is longer on tbe right, sacrificing that extra key.

**Comment by steviefaux** [comment]
So. I was about to raise a bug because I have VERY similad issue. When I'm in other apps the keyboard seems fine with . to the right of the space.

However, I start typing in Firefox and the keyboard layout appears to change with / where the . is. As I type quickly the / frequently pops up and appears to be typed even when I'm nowhere near that key.

I've recorded my key strokes with an Android app on Windows so I can play it back. It looks like I'm typing the / but I know I'm not.

Thought it must be an issue with my typing but it was getting so annoying I switched back to Microsoft's keyboard and I don't have the issue.

So there must be some odd bug in the futo keyboard causing it.
